unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/ChatHeadGroupView.cs

```diff
@@ -38,6 +39,19 @@ public void Initialize(IChatController chatController, IFriendsController friend
         CommonScriptableObjects.rendererState.OnChange += RendererState_OnChange;
     }
 
+    private void Start()
+    {
+        contentParentRT = container.parent as RectTransform;
+        gameObject.SetActive(false);
+    }
+
+    private void Update()
+    {
+        // NOTE: Update is only enabled when SetParentContainerAsDirty is called 
+        DCL.Helpers.Utils.ForceUpdateLayout(contentParentRT, delayed: true);
+        this.enabled = false;
```

Comment:
> Only a question: Why can't we do this directly in the `SetParentContainerAsDirty()` function, instead of here in the `Update()` event?

Is just to avoid the case where multiples chat heads are added and set the layout of dirty. This way we ensure to only call it once and avoid the performance cost of the operation

Reply:
Ok, thanks! you have my approve! :)